contracts/crowdfund/AtomicManualParty.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.20;
+
+import { IGlobals } from "../globals/IGlobals.sol";
+import { LibGlobals } from "../globals/LibGlobals.sol";
+import { IPartyFactory } from "../party/IPartyFactory.sol";
+import { Party } from "../party/Party.sol";
+import { IERC721 } from "../tokens/IERC721.sol";
+import { MetadataProvider } from "../renderers/MetadataProvider.sol";
+import { Proxy } from "../utils/Proxy.sol";
+import { Implementation } from "../utils/Implementation.sol";
+
+/// @title AtomicManualParty
+/// @notice Singleton that is called to create a party manually with an array
+///         of party members and their votes.
+contract AtomicManualParty {
+    error PartyMembersArityMismatch();
+
+    // The `Globals` contract storing global configuration values. This contract
+    // is immutable and it’s address will never change.
+    IGlobals private immutable _GLOBALS;
+    IPartyFactory private immutable _PARTY_FACTORY;
```

Comment:
Since `PartyFactory` can change overtime, should this be read from Globals instead?

Reply:
yeah good idea. We can bring it back to globals